scripts/logserver.js

```diff
@@ -2,7 +2,15 @@
 // replies with 200 Ok. This is very useful when debugging telemetry:
 // env HADRON_METRICS_SEGMENT_API_KEY='ignore' HADRON_METRICS_SEGMENT_HOST='http://localhost:8000' npm start
 const http = require('http');
+const { inspect } = require('util');
 
+function write(payload) {
+  if (process.stdout.isTTY) {
+    console.log(inspect(payload, { depth: Infinity, colors: true }));
```

Comment:
Is there a case in which we want `colors: true` in which we wouldn’t have already gotten color support? `console.dir()` does look at the color support (not just TTY-ness) of the output stream before writing to it to automatically determine the right `colors:` value.

Reply:
This is how I've always done it in the past, I haven't looked closely into how console.dir works.  And I didn't even notice it already calls `inspect` - useful! 

`console.dir` doesn't output valid json though, so I think it still is worthwhile to leave the pipe detection and use JSON.stringify instead